api-overview/about-overview.md

```diff
@@ -2,43 +2,43 @@
 
 ### When Do I Need an Overview?
 
-Every document set should include an overview. This document tells potential users or buyers of the product the things they might want to know before adoption.
+Your API documentation should always include an overview document. This document tells potential users or buyers what they might want to know before deciding to use your API.
```

Comment:
* There is a subtle generalisation introduced here which could be interpreted to be technically incorrect. "Every document set" refers to a set of documentation, and hence justifies an overview. "Your API documentation" might be a single API, and hence need not include an overview.

* I prefer "should" over "should always". "should always" mixes the meaning of "most times" and "always". If you mean "always" then words like "must" and "shall" are usually better. Also best to be concise and use only 1 word instead of 2. It makes the doc easier to understand. However, there are almost always edge cases in documentation. Adding "always" breaks this guidance. From memory, the Google style guide reference to guidance over rules for the same reason.

* I prefer "before adoption" to "before deciding to use your API". Reason: Less words means more concise. Also "adoption" covers more use cases than "deciding to USE your API". It also doesn't lock you to the API use case.

Reply:
I like you suggested change:

* From: buyers of the product the things

* To: buyers what they

Reply:
I've implemented the second two changes, however with regards to "Every document set" I would assume that the people coming to this for guidance wouldn't be those with a single API. Also just to clarify do you mean a single API endpoint? I thought even that should have a quickstart if it isn't a simple public endpoint which most aren't, and since it would have some kind of a reference manual than that would also be a set. So then wouldn't someone having a single API with the need to only have one document be a very edge case we shouldn't focus on?